### PR TITLE
enhancment: fix and polish sidebar toasts

### LINKED ISSUE #2214: ️🛠 - Fix and polish sidebar toasts
### Task description

_No response_

### Requirements

- Remove early access sidebar toast
- Update display logic for backup and autoupdate toast
- Make backup toast smaller

### PR DESCRIPTION
**Pull request by nicole-obrien**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by nicole-obrien** [comment]
draft until after next release

File: packages/desktop/views/dashboard/components/toasts/VersionToast.svelte [hunk 1]
```diff
@@ -10,17 +10,14 @@
...
-    body={localize(`${localeKey}.body`)}
-    open
-    dismissable={false}
-    button={APP_STAGE === AppStage.PROD ? { text: localize(`${localeKey}.button`), onClick } : undefined}
-/>
+{#if APP_STAGE !== AppStage.PROD}
+    <SidebarToast
+        color={TOAST_PROPS[APP_STAGE]}
+        header={localize(`popups.appUpdate.${APP_STAGE}`)}
+        body={localize(`${localeKey}.body`)}
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/components/toasts/VersionToast.svelte]: For this 1 location, we dont need a `localeKey` var

```suggestion
        body={localize('views.dashboard.toasts.version.body')}
```

File: packages/desktop/views/dashboard/components/toasts/VersionToast.svelte [hunk 2]
```diff
@@ -10,17 +10,14 @@
...
 
-<SidebarToast
-    color={TOAST_PROPS[APP_STAGE]}
-    header={localize(`popups.appUpdate.${APP_STAGE}`)}
-    body={localize(`${localeKey}.body`)}
-    open
-    dismissable={false}
-    button={APP_STAGE === AppStage.PROD ? { text: localize(`${localeKey}.button`), onClick } : undefined}
-/>
+{#if APP_STAGE !== AppStage.PROD}
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/components/toasts/VersionToast.svelte]: this check is duplicated here and in `DashboardSidebar`

File: packages/shared/src/locales/en.json [hunk 3]
```diff
@@ -600,12 +600,11 @@
...
                 "backup": {
                     "header": "Backup Required",
-                    "body": "You have not backed up your profile yet! We highly recommend backing this up in case you lose access to this device.",
+                    "body": "You haven't backed up your profile in while.",
                     "button": "Backup now"
                 },
                 "version": {
-                    "body": "You are currently using an early version of Bloom.",
-                    "button": "Find out more"
+                    "body": "You are currently using an development version of Bloom."
```
Comment (MarkNerdi) [inline packages/shared/src/locales/en.json]:
```suggestion
                    "body": "You are currently using a development version of Bloom."
```

**Review by MarkNerdi — APPROVED** [review]

**Pull request merged**